core/src/main/scala/codesearch/core/db/DefaultDB.scala

```diff
@@ -2,20 +2,26 @@ package codesearch.core.db
 
 import java.sql.Timestamp
 
+import codesearch.core.index.repository.SourcePackage
 import codesearch.core.model._
-import slick.lifted.TableQuery
 import slick.jdbc.PostgresProfile.api._
+import slick.lifted.TableQuery
 
-import scala.concurrent.Future
 import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
```

Comment:
Can we make scalafmt sort imports automatically?

Reply:
We already have it. We use `asciisortimports`.
More patterns here: https://scalameta.org/scalafmt/#SortImports